CMakeLists.txt

```diff
@@ -41,8 +41,15 @@ if(VTKm_FOUND)
   list(APPEND _libs vtkm_cont)
 endif()
 
-add_library(stem
-  ${_src})
+if(TARGET vtkm::cuda)
+  # Compile with CUDA
+  vtkm_compile_as_cuda(cudaSource ${_src})
+  add_library(stem ${cudaSource})
+  target_compile_definitions(stem PRIVATE
```

Comment:
Setting the default device adapter isn't needed anymore, and is a legacy define we are currently removing.